src/WebJobs.Script/Utility.cs

```diff
@@ -447,6 +448,20 @@ internal static bool IsSingleLanguage(IEnumerable<FunctionMetadata> functions, s
             return true;
         }
 
+        internal static bool ShouldInitiliazeLanguageWorkers(IEnumerable<FunctionMetadata> functions, string language)
+        {
+            if (!string.IsNullOrEmpty(language) && language.Equals(LanguageWorkerConstants.DotNetLanguageWorkerName, StringComparison.OrdinalIgnoreCase))
+            {
+                return false;
+            }
+            if (functions != null && functions.Any())
+            {
+                List<string> dotNetLanguages = new List<string>() { DotNetScriptTypes.CSharp, DotNetScriptTypes.FSharp, DotNetScriptTypes.DotNetAssembly };
+                return functions.Where(f => !dotNetLanguages.Contains(f.Language, StringComparer.OrdinalIgnoreCase)).Any();
```

Comment:
Remove the where clause and just pass the lambda into the `Any` call. Should actually be able to simplify this by using `All` instead (avoid the negation)